frontend/coprs_frontend/coprs/whoosheers.py

```diff
@@ -54,13 +54,18 @@ def update_package(cls, writer, package):
 
     @classmethod
     def insert_copr(cls, writer, copr):
+        # Avoid indexing the largest projects to reduce the whooshee index size,
+        # and thus memory usage
+        packages=cls.get_package_names(copr)
+        if len(packages) > 10000:
+            packages = []
```

Comment:
How much does this help?  I'm not against it, just please document it.

Reply:
Documented in the commit description, on my laptop it went from 99M to 30M. Not sure if it is worth it but I'd say yes.

Reply:
Ah, thanks.  What makes me curious now is how **just the set of package names** can make the cache 3 times bigger?  Tested this:
```
coprdb=# select sum(ARRAY_LENGTH(REGEXP_SPLIT_TO_ARRAY(description || ' ' || instructions, '\s'), 1)) from copr;
┌─────────┐
│   sum   │
├─────────┤
│ 2502387 │
└─────────┘
(1 row)

coprdb=# select count(*) from package;
┌─────────┐
│  count  │
├─────────┤
│ 1619863 │
└─────────┘
(1 row)
```

Reply:
This means that packages from PyPI and Rubygems project cannot be found. :(

Reply:
I removed that commit, I think we don't need it.